test/metabase/api/login_history_test.clj

```diff
@@ -58,7 +58,7 @@
                  :device_description "Browser (Chrome/Windows)"
                  :ip_address         "185.233.100.23"
                  :active             true
-                 :location           "Begles, Nouvelle-Aquitaine, France"
+                 :location           "Talence, Nouvelle-Aquitaine, France"
```

Comment:
This change is to fix https://app.circleci.com/pipelines/github/metabase/metabase/16017/workflows/4e935ffe-050c-407c-9d25-c9e0998d9ae7/jobs/616408